spec/system/admin/projects_show_spec.rb

```diff
@@ -93,9 +93,10 @@
           within("table.js--transactions-table") do
             click_link originating_order_facility1.id.to_s
           end
-        
 
-          expect(page).to have_content(originating_order_facility1.id.to_s)
+          within("table#order-management") do
+            expect(page).to have_content(originating_order_facility1.id.to_s)
```

Comment:
Is it here that it fails or is it when it tries to click? I was under the impression the problem was when trying to click, as here we're only checking that it has content but we don't have a number of matches

Reply:
There's another failure where there can be multiple links with the same text (like this https://github.com/wyeworks/nucore-open/actions/runs/13789002261/job/38563725814) but I'm still looking into it

Reply:
I think I fixed a bunch of them by using the `href` option with the link I expected to be there. If I'm not mistaken, the problem is when Order and Order detail have the same id